lib/elastomer_client/client/bulk.rb

```diff
@@ -280,9 +280,9 @@ def create(document, params)
       # params   - Parameters for the update action (as a Hash) (optional)
       #
       # Examples
-      #   update({"foo" => "bar"}, {:_id => 1}
-      #   update({"foo" => "bar"}, {:id => 1}
-      #   update("foo" => "bar", "_id" => 1)
+      #   update({"doc" => {"foo" => "bar"}}, {:_id => 1}
```

Comment:
Do you know if this is proper syntax for ES5 and ES8? I just want to make sure that if it's only correct for one of them, maybe we keep both and specify which version they each work on

Reply:
I double checked through the ES docs and confirmed with myself this is good! Thanks for double checking though 😄